front/app/modules/commercial/widgets/admin/containers/index.tsx

```diff
@@ -1,22 +1,40 @@
-import React, { PureComponent } from 'react';
+import React from 'react';
 import IdeasWidget from './IdeasWidget';
 import { SectionTitle, SectionDescription } from 'components/admin/Section';
+import { Box, colors } from '@citizenlab/cl2-component-library';
+import { useLocation } from 'react-router-dom';
+import clHistory from 'utils/cl-router/history';
+import GoBackButton from 'components/UI/GoBackButton';
 
 import messages from '../messages';
 import { FormattedMessage } from 'utils/cl-intl';
 
-export default class AdminSettingsWidgets extends PureComponent {
-  render() {
-    return (
-      <>
-        <SectionTitle>
-          <FormattedMessage {...messages.titleWidgets} />
-        </SectionTitle>
-        <SectionDescription>
-          <FormattedMessage {...messages.subtitleWidgets} />
-        </SectionDescription>
+const AdminSettingsWidgets = () => {
+  const location = useLocation();
+
+  const goBack = () => {
+    clHistory.goBack();
```

Comment:
I need to click Go back twice on the Widget page.

Reply:
That is because of the errors in console. The browser back button should give you the same behavior. This won't be the case in production so we can ignore